### PR TITLE
Docs update #3

### PR DESCRIPTION
**Pull request by vlada-shubina**
### Problem
Updating docs in the repo / Wiki

### CODE REVIEW

File: docs/Exit-Codes.md [hunk 1]
```diff
@@ -166,19 +166,19 @@ foobarbaz could not be installed, no NuGet feeds are configured or they are inva
 For details on current exit code please visit https://aka.ms/templating-exit-codes#103
 ```
 
-## <a name="104"></a>104 - PostAction operation was cancelled
+## <a name="104"></a>104 - post action operation was cancelled
```
Comment (JanKrivanek) [inline docs/Exit-Codes.md:169]:
```suggestion
## <a name="104"></a>104 - Post action operation was cancelled
```

All other codes headers start with capital

File: docs/Home.md [hunk 2]
```diff
@@ -9,8 +9,12 @@
...
\ No newline at end of file
+- [Reference for `template.json`](Reference-for-template.json.md)
+- [Post actions](Post-Action-Registry.md)
+- [Symbols Generators](Available-Symbols-Generators.md)
+- [Value Forms](Value-Forms.md)
+- [Conditional content generation](Conditional-processing-and-comment-syntax.md)
+- [Template constraints](Constraints.md)
+- [Binding and project context evaluation](Binding-and-project-context-evaluation.md)
+- [Authoring tools: localization](authoring-tools/Localization.md)
+- [Authoring tools: testing the templates](authoring-tools/Templates-Testing-Tooling.md)
```
Comment (JanKrivanek) [inline docs/Home.md:20]: Does the wiki play well with the `.md` extension? But we should likely get rid of the wiki anyways

File: docs/Reference-for-template.json.md [hunk 3]
```diff
@@ -1,55 +1,101 @@
...
+        "type": "host",
+        "args": [
+            {
+                "hostname": "dotnetcli",
+                "version": "[6.0.100, )"
+            }
+        ]
+    }
+}
+```
```
Comment (JanKrivanek) [inline docs/Reference-for-template.json.md:70]:
```suggestion
```json
"constraints": {
  "linux-only": {
    "type": "os",
    "args": "Linux"
  },
  "sdk-only": {
    "type": "host",
    "args": [
      {
        "hostname": "dotnetcli",
        "version": "[6.0.100, )"
      }
    ]
  }
}
```  
```

We have idention of 2 spaces in other json examples

File: docs/Reference-for-template.json.md [hunk 4]
```diff
@@ -201,46 +247,46 @@ String optional parameter, replaces TargetFrameworkOverride:
...
+        "choice": "android",
+        "description": "android mobile"
+      },
+      {
+        "choice": "nix",
+        "description": "Linux distributions"
+      }
+    ],
+    "defaultValue": "MacOS|iOS"
+  }
```
Comment (JanKrivanek) [inline docs/Reference-for-template.json.md:285]: Mix of indentions in this sample

File: docs/Reference-for-template.json.md [hunk 5]
```diff
@@ -265,7 +311,7 @@ A symbol that defines transformation of another symbol.  The value of this symbo
 |Name|Description|Mandatory|
 |---|---|---|
 |`type`|`derived`| yes |
-|`valueSource`|The name of the other symbol whose value will be used to derive this value.| yes |
+|`valueSource`|The name of the other symbol which value will be used to derive this value.| yes |
```
Comment (JanKrivanek) [inline docs/Reference-for-template.json.md:314]: I belieive which is for singular

**Review by JanKrivanek — APPROVED** [review]
Great improvements! Looks good... I have few nit-picky comments :-) but it's already good even as is.

File: docs/Reference-for-template.json.md (hunk 5 above)
Comment (vlada-shubina) [reply docs/Reference-for-template.json.md:314]: I think `whose` can only be applied to a person, but not sure.

Comment (JanKrivanek) [reply docs/Reference-for-template.json.md:315]: You are indeed correct.
You can disregard :)

**Pull request merged**